CMakeLists.txt

```diff
@@ -406,6 +406,10 @@ option(SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY
   "Enable experimental Swift concurrency model"
   FALSE)
 
+option(SWIFT_ENABLE_COMPATIBILITY_OVERRIDES
```

Comment:
This seems to be specific to the stdlib would you mind just moving this to `stdlib/CMakeLists.txt` please?